Medias.postman_collection.json

```diff
@@ -0,0 +1,71 @@
+{
+	"info": {
+		"_postman_id": "051920ce-6252-422b-97e8-c15e37ed5695",
+		"name": "Medias",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Create a media",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "author",
+							"value": "zaher",
+							"type": "text"
+						},
+						{
+							"key": "files[1][platform_id]",
+							"value": "2",
+							"type": "text"
+						},
+						{
+							"key": "files[1][file]",
+							"type": "file",
+							"src": "/Users/z.hassan/Downloads/Frame 2.png"
```

Comment:
how is this PATH supposed to work for someone trying this collection ?

Reply:
He will need to update with a file from his file system. I can't put an env variable as a file.